lib/Upcloud/ServerApi.php

```diff
@@ -421,13 +421,14 @@ public function deleteFirewallRuleAsyncWithHttpInfo(string $serverId, float $fir
      * Delete server
      *
      * @param string $serverId Id of server to delete (required)
+     * @param bool $deleteStorage Delete attached storage devices
      * @throws ApiException on non-2xx response
      * @throws InvalidArgumentException|GuzzleException
      * @return void
      */
-    public function deleteServer(string $serverId): void
+    public function deleteServer(string $serverId, bool $deleteStorage = false): void
```

Comment:
Might be better to have this param named as `$deleteAttachedStorageDevices` since using this boolean will delete all of the storage devices attached to server.

Reply:
Thanks @PopoSensei, I've made this update.